source/chapters/1/intro.md

```diff
@@ -25,31 +25,40 @@ However, if you have errors, check if you ran the commands correctly.
 Afterwards, Google is your best friend! 
 
 On Windows, simply download the installer and follow the basic Windows installation procedure that most software has.
-Here, instead of Bash, Terminator or iTerm, you can use Powershell or the good old Command Prompt.
-The commands are mostly the same.
 
 ##Tools
 The tool that works best for us, and the one which we recommend you use, would be WebStorm, from Jetbrains.
 Students can get it for free, and other people can get a free 30-day evaluation trial.
-You can download Webstorm from [here](https://goo.gl/gqmqwt).
+You can download Webstorm from [here](https://www.jetbrains.com/webstorm/download/#section=linux).
 
 To learn more about Webstorm go [here](https://www.jetbrains.com/webstorm/documentation/).
 
-Another tool which is quite easy to learn and use would be Sublime Text. 
-This is a code editor with a lot of features that will make your life a whole lot easier, 
-if you're looking to get in the coding business.
+A free alternative to Webstorm would be Sublime Text. This is an extensible, feature-rich code editor, that is 
+relatively easy to learn.  
+It will make your life a whole lot easier, if you're looking to get in the coding business.
 You can download it from [here](https://www.sublimetext.com/3).
 
 To learn how to use it properly,go [here](https://scotch.io/bar-talk/the-complete-visual-guide-to-sublime-text-3-getting-started-and-keyboard-shortcuts).
 
+To be more specific, we recommend the following plugins for Meteor development with Sublime Text: 
+-Meteor Snippets
+-Meteor Autocomplete (TernJS) 
+
 ## Creating a project
 
-After you have installed Meteor, you can easily create a new project by running the following command in your terminal:
+After you have installed Meteor, you can easily create a new project by running the following command in 
+your terminal:
 
 ```
 meteor create myProjectName
 ```
 
+![Windows](../images/windowsSmall.png "Windows icon")
+If you get any errors when trying to create a Meteor project in Windows, download the NodeJs installer from 
```

Comment:
I feel like this creates confusion for users,

Reply:
Explained what happens during these steps.